api/registration/urls.py

```diff
@@ -18,5 +21,12 @@
     path('password-reset/', include('django_rest_passwordreset.urls', namespace='password-reset')),
 
     path('verify-email/', VerifyEmailView.as_view(), name='verify-email'),
+
+
+    # endpoints
```

Comment:
I'm trying to understand how the noted is relevant to the appointments but I think you mixed between two tasks

Reply:
thanks i just noticed